packages/video_player/video_player_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.6.4
+
+* Extracts `FVPVideoPlayer` and `FVPFrameUpdater` from the `FVPVideoPlayerPlugin.m` file and places them into separate files.
```

Comment:
> i don't think this requires a version bump

It does, because it changes production code.

> since no behavioral change for users.

You mean no **intended** behavioral change for users.

The reason the version policy does not have an exemption for "this isn't supposed to change any behavior" is that what people intend and what actually happens are not always the same thing, and batching a change that "doesn't change behavior for users" (but actually, does) with some random later change makes debugging and rolling back breakage much harder.

(Whereas if it actually doesn't change any behavior, as intended, publishing it anyway is harmless.)

Reply:
The changelog description doesn't need to be this specific about implementation details though. "Refactors native code structure." or something like that is all we need to say.

Reply:
Updated the changelog